server/src/models/Order/Order.ts

```diff
@@ -0,0 +1,13 @@
+import { model, Schema } from "mongoose";
+import IOrder from "./IOrder";
+
+const OrderSchema: Schema = new Schema({
+  user: { type: Schema.Types.ObjectId, ref: "User" },
+  products: [{ type: Schema.Types.ObjectId, ref: "Product" }],
+  total: { type: Number, default: 0, required: true },
+  address: { type: String, required: true },
+  coupon: { type: String, required: true },
```

Comment:
Can we remove required: true from coupon and address? Not all orders are required to have coupons, or have a product delivered to address